enterprise/service/license.go

```
@@ -138,15 +130,13 @@ func (s *licenseService) writeLicense(token string) error {
 	return ioutil.WriteFile(s.config.StorePath, []byte(token), 0644)
 }
 
-func validPlanType(candidate string) error {
-	for _, plan := range validPlans {
-		if plan == candidate {
-			return nil
-		}
+func convertPlanType(candidate string) api.PlanType {
```

I suggest to return error if plan is an invalid value instead of returns FREE